questdb-rs/src/ingress/mod.rs

```diff
@@ -1011,6 +1069,175 @@ impl Buffer {
         Ok(self)
     }
 
+    /// Record a multidimensional array value for the given column.
+    ///
+    /// Supports arrays with up to [`MAX_ARRAY_DIMS`] dimensions. The array elements must
+    /// implement [`ArrayElement`] trait which provides type-to-[`ElemDataType`] mapping.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage with direct dimension specification:
+    ///
+    /// ```
+    /// # #[cfg(feature = "ndarray")]
+    /// # {
+    /// # use questdb::Result;
+    /// # use questdb::ingress::Buffer;
+    /// # use ndarray::array;
```

Comment:
Don't comment the `use ndarray::array` line here. Should be shown as part of the example.